packages/sdk/src/AbstractConfigManager.ts

```diff
@@ -322,7 +326,7 @@ export abstract class AbstractConfigManager {
         } catch (err) {}
     }
 
-    private async validateConfig(newConfig: ISshConfigExt): Promise<ISshConfigExt | undefined> {
+    private async validateConfig(newConfig: ISshConfigExt, askForPassword = true): Promise<ISshConfigExt | undefined> {
```

Comment:
I may not be fully grasping the implications of defaulting this to `true`, but are we forcing the SDK to be always interactive? 
I'm mostly concerned of pipelines and other non-interactive ways of using the SDK/CLI 😋